### DISCUSSION TITLE
validation data ipp portal data upload

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#99] [General]
I get the following files for the validation data in the `checkpoint/experiment_1`, which file do I need to upload on ipp portal (in the attached pic) to see the performance 

`-rw-rw----. 1 irfan21h project_2004598 112M Jul 15 23:02 best_model.pkl

-rw-rw----. 1 irfan21h project_2004598  43G Jul 16 02:23 aggregator_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.3G Jul 16 02:24 14_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.3G Jul 16 02:24 15_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:24 03_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.3G Jul 16 02:25 21_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.4G Jul 16 02:25 17_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:25 04_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:25 16_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:26 11_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:26 13_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:26 20_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:27 05_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.3G Jul 16 02:27 08_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:27 06_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.3G Jul 16 02:28 02_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.3G Jul 16 02:28 19_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:29 12_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:29 10_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:29 09_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:29 01_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.0G Jul 16 02:30 22_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.2G Jul 16 02:30 07_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598 5.0G Jul 16 02:30 18_tensor_db.pkl
-rw-rw----. 1 irfan21h project_2004598  19K Jul 16 02:30 state.pkl
-rw-rw----. 1 irfan21h project_2004598 112M Jul 16 02:30 temp_model.pkl
drwxrws---. 2 irfan21h project_2004598  12K Jul 16 02:47 model_outputs`

<img width="962" alt="Screenshot 2021-07-16 at 3 24 02" src="https://user-images.githubusercontent.com/46925837/125873784-d275bb94-ced3-412e-89c3-a8898edb6a74.png">

**Comment by sarthakpati** [comment]
I believe it would be the segmentation files under model outputs. But ensure that there is no `_seg` present in the filenames.

**Reply by dskhanirfan** [reply]
There are 125 files in model outputs with names like FeTS21_Validation_001.nii.gz, Do I compress the model outputs folder and upload it or should I upload all the files directly ?

**Reply by sarthakpati** [reply]
I think either should work. But you can upload all the fes directly.

**Reply by dskhanirfan** [reply]
I was not able to download the data from model_outputs, I got the following error
`scp: experiment_1/model_outputs: not a regular file`

**Reply by sarthakpati** [reply]
It is a directory, so you need to do `scp -r`

**Reply by dskhanirfan** [reply]
ok. how long does it take to get the results from ipp portal?

**Reply by sarthakpati** [reply]
Usually within a couple of days.
